### ISSUE TITLE
Error on MMU2 filament load fail

### ISSUE THREAD

**Issue by Zunth5** [prusa3d/Prusa-Firmware#2685]
Please, before you create a new bug report, please make sure you searched in open and closed issues and couldn't find anything that matches.

**Printer type** - MK2.5S
**Printer firmware version**-  3.8.1 (updated to the latest yesterday) I think its 3.8.1

**MMU Upgrade** - MMU2S
**MMU upgrade firmware version 1.0.6

**Describe the bug**
3rd load failed with cutter during 2 filament mmu print. Pressed select button mk2.5s maybe five seconds after cooldown started. Resulted in the m112 called. Printing using octoprint. Reset mmu multiple times in the same print without issue.

![FF4F3DB9-0A0E-4710-A39A-7841D4085C4C](https://user-images.githubusercontent.com/6885957/82760649-f2be5e80-9dc2-11ea-917e-23c3caa245c8.jpeg)

  
**To Reproduce**
  Basically above
  
**Expected behavior**
Reheat and not to trigger an estop
  
**G-code**
  I dont have it at the moment or logs :/ working from my mobile device at a family location over the holiday weekend.

**Video**
  Please attach a video. It usually helps to solve the problem.

**Comment by awenelo** [comment]
The printer can not directly trigger an M112. The only way for an M112 to occur is for it to read it in a Gcode file or have it be sent from octoprint. Can you go into the octoprint settings and in `Serial connection -> Behaviour -> Error handling` and uncheck "Send `M112` on disconnect due to error" and then try to cause the error again? The most likely thing that happened here was that the and error was triggered, causing octoprint to disconnect, sending and `M112`.
